EMS-UI/src/environments/environment.prod.ts

```diff
@@ -1,8 +1,7 @@
 export const environment = {
   production: true,
-  EMS_API: '/api',
+  EMS_API: 'https://ems.cactus-staging.iucdspilot.uk',
```

Comment:
Since we'll have a staging and prod version of the UI, we may need to have separate env files for each, or set them from the environment at startup of the image. Can you log a ticket to handle this when we get to production infra?

Reply:
Good point, [raised](https://jira.digital.nhs.uk/browse/CDSCT-142).